src/main/java/seedu/address/model/note/Note.java

```diff
@@ -2,15 +2,20 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.HashSet;
 import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
 
 
 /**
  * Generic Note class for country and client notes.
  */
 public class Note {
-    public static final String MESSAGE_CONSTRAINTS = "message contraints for note";
+    public static final String MESSAGE_CONSTRAINTS = "message constraints for note";
     protected final String noteContents;
+    protected final Set<Tag> tags = new HashSet<>();
```

Comment:
why are these `protected`? I couldn't find any other classes in the same package using them